WooCommerce/TodayStatsWidget/TodayViewController.swift

```diff
@@ -0,0 +1,366 @@
+            return
+        }
+
+        syncSiteStats(timeRange: .today) { [weak self] (result) in
+            switch result {
+            case .failed:
+                self?.sections = [Section(rows: [.errorMessage])]
+                completionHandler(.failed)
+                return
+            case .newData:
+                self?.sections = [Section(rows: [.todayStats])]
+                completionHandler(.newData)
+                return
+            case .noData:
+                self?.sections = [Section(rows: [.todayStats])]
+                completionHandler(.noData)
+                return
+            default:
+                return
+            }
+        }
+    }
+
+    // Sync remotely all the stats showed in the widget
+    func syncSiteStats(timeRange: StatsTimeRangeV4,
+                            onCompletion: @escaping (NCUpdateResult) -> Void) {
+
+        guard let credentials = credentials else {
+            onCompletion(.failed)
+            return
+        }
+        guard let site = site else {
+            onCompletion(.failed)
+            return
+        }
+
+        var tempStats = StatsData()
+
+        let network = AlamofireNetwork(credentials: credentials)
+
+
+        /// Calculation of dates
+        let dateFormatter = DateFormatter.Defaults.iso8601WithoutTimeZone
+        dateFormatter.timeZone = site.siteTimezone
+        let earliestDate = dateFormatter.string(from: Date().startOfDay(timezone: site.siteTimezone))
+        let latestDate = dateFormatter.string(from: Date().endOfDay(timezone: site.siteTimezone))
+
+        let quantity = timeRange.siteVisitStatsQuantity(date: Date(), siteTimezone: site.siteTimezone)
+
+        let group = DispatchGroup()
+
+        /// Load Order Stats
+        group.enter()
+        let remoteOrderStats = OrderStatsRemoteV4(network: network)
+        remoteOrderStats.loadOrderStats(for: site.siteID,
+                                        unit: timeRange.intervalGranularity,
+                                        earliestDateToInclude: earliestDate,
+                                        latestDateToInclude: latestDate,
+                                        quantity: quantity) { [weak self] (orderStatsV4, error) in
+            guard error == nil else {
+                if let error = error as? DotcomError, error == .noRestRoute {
+                    self?.error = .statsV4NotAvailable
+                }
+                else {
+                    self?.error = .errorFetchingStats
+                }
+                group.leave()
+                return
+            }
+            tempStats.totalOrders = orderStatsV4?.totals.totalOrders
+
+            // TODO: implement currency formatter
+            // let currencyCode = CurrencySettings.shared.symbol(from: CurrencySettings.shared.currencyCode)
+            // totalRevenueText = CurrencyFormatter().formatHumanReadableAmount(String("\(orderStats.totals.grossRevenue)"), with: currencyCode) ?? String()
```

Comment:
nit: maybe these can be added at the next PR since we generally don't keep commented out / unused code in the codebase unless it's being worked on soon? 🙂